src/components/PopoverButton/index.spec.tsx

```diff
@@ -1,53 +1,72 @@
-import { shallow } from 'enzyme';
 import * as React from 'react';
+import { Store } from 'redux';
 import { Button, Overlay, OverlayChildrenParams } from 'react-bootstrap';
 
-import { createFakeRef } from '../../test-helpers';
+import { createFakeRef, shallowUntilTarget, spyOn } from '../../test-helpers';
```

Comment:
Nit: I try to keep the imports in alphabetical order (by source), but it's not a big deal.

Reply:
What part of it isn't alphabetical?

Reply:
Do you mean that this line should come after `../../reducers...`?

Reply:
I could make that change but I don't see how it's very helpful because if you're looking for something, you're more likely looking for an exported object rather than the file itself.

Reply:
I would see them in this order, alphabetically:

```
import configureStore from '../../configureStore';
import { actions as popoverActions } from '../../reducers/popover';
import { createFakeRef, shallowUntilTarget, spyOn } from '../../test-helpers';
```

because `c` comes before `r` comes before `t`. But as I said, it's not a big deal.